modules/rte_mis_lottery/src/Plugin/DevelGenerate/MiniNodeDevelGenerates.php

```diff
@@ -313,26 +346,26 @@ private function createMiniNode($type = ''): void {
           'field_student_application_number' => $this->generateRandomString(11),
           'field_location' => $this->generateRandomSelectValue('field_location'),
           'field_date_of_birth' => date('2019-01-01'),
-          'field_gender' => ['transgender'],
+          'field_gender' => ['transgender', 'boy', 'girl'],
```

Comment:
Can we fetch this from the configuration itself?

Reply:
This file is only for test purpose. We can ignore this for now.